atlasdb-cassandra/src/test/java/com/datastax/driver/core/CassandraTokenRanges.java

```diff
@@ -37,7 +38,7 @@ public static TokenRange create(Token start, Token end) {
     }
 
     public static Token getToken(String hexBinary) {
-        return FACTORY.hash(ByteBuffer.wrap(DatatypeConverter.parseHexBinary(hexBinary)));
+        return FACTORY.hash(ByteBuffer.wrap(HEX.decode(hexBinary)));
```

Comment:
I am not sure there are any guarantees that the input string is already lowercase. The JAXB DatatypeConverter handles both upper and lower hex encoding, but Guava does not so we may need to convert case prior to hex conversion. If we want to avoid the allocation conversion, we could do something like https://github.com/palantir/atlasdb/pull/6468

```suggestion
        return FACTORY.hash(ByteBuffer.wrap(HEX.decode(hexBinary.toLowerCase(Locale.ROOT))));
```

https://github.com/javaee/jaxb-spec/blob/b25f7217b22d0ff17e6f160f28b1521399fc42f6/jaxb-api/src/main/java/javax/xml/bind/DatatypeConverterImpl.java#L452-L483

Reply:
Note that this is test code and this code path is only hit from these 3 statements.

https://github.com/palantir/atlasdb/blob/0e5f3fbe8d90e3d00c4d2330a1606570f40173ec/atlasdb-cassandra/src/test/java/com/palantir/atlasdb/cassandra/backup/CqlMetadataTest.java#L52-L54

Reply:
I just took this opportunity to eliminate the `jaxb` dependency entirely, since we replace this with a dependency we already have.